src/accumulator.jl

```diff
@@ -92,7 +90,9 @@ Decrements the count for `x` by `v` (defaulting to one)
 dec!(ct::Accumulator, x, v::Number) = (ct[x] -= v)
 dec!(ct::Accumulator{T,V}, x) where {T,V} = dec!(ct, x, one(V))
 
-#TODO: once we are done deprecating `pop!` for `reset!` then add `pop!` as an alias for `dec!`
+Base.pop!(ct::Accumulator, x) = dec!(ct, x)
+Base.pop!(ct::Accumulator, x, default) = haskey(ct, x) ? dec!(ct, x) : default
```

Comment:
similarly to what I mentioned above, here I would have expected checking for "value equal zero", not only "has key"